### PR TITLE
V2 update - Fixes #3

### LINKED ISSUE #3: v2-update

### PR DESCRIPTION
**Pull request by mayuir2014**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -645,14 +645,14 @@ the "copyright" line and a pointer to where the full notice is found.
...
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see {http://www.gnu.org/licenses/}.
 
 Also add information on how to contact you by electronic and paper mail.
 
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    {project}  Copyright (C) {year}  {fullname}
+    adapt-hello-world  Copyright (C) 2013  Chris Jones
```
Comment (dancgray) [inline LICENSE]: Can you change this line to reflect this repo.

**Comment by dancgray** [comment]
Does this only work with 4 items?

File: LICENSE (hunk 1 above)
Comment (mayuir2014) [inline LICENSE]: It doesn't look like this value has changed for any of the components I looked at, including:

- adapt-pasmenu
- adapt-introcover-menu
- adapt-icon-popup
- adapt-custom-hotgrid

If we come up with the standard that everyone is happy with, everyone can edit this in each of the repos.

**Comment by mayuir2014** [comment]
Yes, it was tweaked to do this initially.

I've added another issue (#4) - 'Allow multiple items in a row option', which will extend the capability to allow 2, 3 or 4 items in a row.

**Pull request merged**